site/book/05-developing-functions/02-developing-in-Go.md

```diff
@@ -13,121 +12,161 @@ the following features:
   manipulating comments; however, it comes at the cost of complexity compared to
   representing resources as idiomatic data structures.
 
+## Prerequisites
+
+#### Install kpt
+
+`kpt` is used to get the example Go projects "get-started" and provides an easy way to generate the function specification
+object `ResourceList`.
```

Comment:
I couldn't parse this sentence. Since `kpt` is required for executing functions anyway, maybe we can skip this explanation and simply say: kpt will be used to test the function that we will be developing, so please use [the instructions]() to install `kpt`. WDYT ?